tests/bats/test-bats-dockerimage.bats

```diff
@@ -2,7 +2,7 @@
 
 @test "With no cmd/args, the image return docker-compose version" {
 	result="$(docker run ${DOCKER_IMAGE_NAME})"
-	[ "$result" == "docker-compose 1.2.0" ]
+	[[ "$result" == *"docker-compose version: 1.3.0"* ]]
```

Comment:
Because it now returns 
    
    docker-compose version: 1.3.0
    CPython version: 2.7.9
    OpenSSL version: OpenSSL 1.0.1e 11 Feb 2013

I though double square brackets were needed for checking with stars, but I might have made a mistake.